model/sequence_file.py

```diff
@@ -40,9 +38,6 @@ def get(self, key):
             ret_val = self._properties_dict[key]
         return ret_val
 
-    def get_files_size(self):
```

Comment:
This became dead code because we can get the file size from the actual files to be more accurate.